community/kernel/src/main/java/org/neo4j/kernel/impl/storageengine/impl/recordstorage/StoreStatement.java

```diff
@@ -308,4 +308,10 @@ public Properties properties()
     {
         return propertyStore;
     }
+
+    @Override
+    public long nextGraphPropertyReference()
```

Comment:
This method is confusingly named imho, which I guess is because of `MetaDataStore.getGraphNextProp`. I think it always returns the reference to the first graph property, so maybe just `getGraphPropertyRefernce` would be better?

Reply:
or maybe even `getGraphPropertyReference` :)